Containerized proxy - copying images to internal registry (#1981)
uyuni-project/uyuni-docs: modules/installation-and-upgrade/pages/containerized-proxy-wo-internet.adoc

[[containerized-proxy-setup-without-internet]]
= Containerized proxy deployment without an internet connection

It is possible to deploy containerized {productname} Proxy in an environment without an internet connection.
In such case, the images are copied to an internal registry, or saved to a tar file.

> This procedure allows re-use of the existing minion FQDN to create a configuration to start a containerized proxy.

Reviewer: this is not related with a proxy deployment without internet connection

> The user should first register the minion to suse manager server, and then re-use the same FQDN in a generation.

Author: ```suggestion
The user should first register the minion to {productname} server, and then re-use the same FQDN in a generation.
```

> Make sure the default port is set up to `8022` for virtualized proxy when creating configuration using [literal]`spacecmd`.

Reviewer: this is not related to connected setup, but rater related to general installation.

The product that is installed in that machine will be SUSE Manager Proxy and will not have the needed packages.
Changing the base channel to have the SUSE Manager client tools can lead to package conflicts and port conflicts.

[IMPORTANT]
====
This procedure applies to {salt} proxies only.
====

[[from.suma.to.internal.reg.without.ssl]]

> == Copy images from suse manager registry to internal registry without SSL configured

Reviewer: ```suggestion
== Copy images from SUSE registry to internal registry without SSL configured
```

[NOTE]
====
If you want to copy the images to a registry with SSL, remove the property [literal]`--dest-tls-verify=false`.
====

> . In a machine with access to "registry.suse.com" (CONFIRM: what machine - SUMA server?) install [literal]`skopeo` by using the command

Author: ```suggestion
. In a machine with access to "registry.suse.com" install [literal]`skopeo` by using the command
```

> . In a machine with access to "registry.suse.com" (CONFIRM: what machine - SUMA server?) install [literal]`skopeo` by using the command

Reviewer: can be any machine, no need to be on suse manager machine

Author: @rjmateus - would this sentence work better:
